celer/dropin_sklearn.py

```diff
@@ -24,16 +22,22 @@ class Lasso(Lasso_sklearn):
 
     The optimization objective for Lasso is::
 
-    (1 / (2 * n_samples)) * ||y - X w||^2_2 + alpha * \sum_j weights_j |w_j|
```

Comment:
keep the Lasso class as it was and add an `ElasticNet` class. This duplicates code but from the user perspective it's much clearer to expose both Lasso and ElasticNet